### ISSUE TITLE
Get memo data from Transaction pool (horizon)

### ISSUE THREAD

**Issue by codehakase** [zulucrypto/stellar-api#5]
Is there a method to stream incoming transaction sorted by a single memo?

Example:

```php
/**
* getTransaction consists of two parameters
* 1. account id (receiver)
* 2. sender wallet (account id)
**/

$transactionStream = $horizon->getTransactions("xxx", "xxxx");

// check memo equals correct data
if ($transactionStream->memo  = "ABCDEFG") {
  // DO something here
}
```

**Comment by zulucrypto** [comment]
Hello @codehakase - yes, you should be able to do this by streaming the Transactions endpoint. I don't have an example in the codebase yet, but I'll include this one with the next release:

```
<?php

require '../vendor/autoload.php';

use \ZuluCrypto\StellarSdk\Horizon\ApiClient;
use \ZuluCrypto\StellarSdk\Model\Transaction;

$client = ApiClient::newPublicClient();

$client->streamTransactions('now', function(Transaction $transaction) {
    printf('[%s] Transaction #%s with memo %s' . PHP_EOL,
        (new \DateTime())->format('Y-m-d h:i:sa'),
        $transaction->getId(),
        $transaction->getMemo()
    );
});
```

The `if` statement in your example above would go right after the printf and look something like:

```
if ($transaction->getMemo() == "ABCDEFG") {
    // ...
}
```

**Comment by codehakase** [comment]
@zulucrypto That'll be great! thanks

**Comment by zulucrypto** [comment]
Closing since example is now available in 0.3.0
